### PR TITLE
Add get_property and get_value to MiotDevice

### PR DESCRIPTION
**Pull request by BenjaminFaal**
Instead of always getting all properties now you can just get one and also get the value directly

### CODE REVIEW

File: miio/miot_device.py [hunk 1]
```diff
@@ -29,6 +29,16 @@ def get_properties_for_mapping(self) -> list:
...
         )
+    
+    def get_property(self, property_key: str) -> dict:
+        property = {"did": property_key, **self.mapping.get(property_key)}
+
+        return self.get_properties(
+            [property], property_getter="get_properties", max_properties=1
+        )[0]
+
+    def get_value(self, property_key: str):
```
Comment (rytilahti) [inline miio/miot_device.py:40]: Would it be better just to have only `get_property` returning directly the value? Please also add docstring(s).

Comment (BenjaminFaal) [reply miio/miot_device.py:41]: get_property returns the whole property including the `code` property so you can check the code for the status of the property
get_value returns the value directly

File: miio/miot_device.py [hunk 2]
```diff
@@ -29,6 +29,18 @@ def get_properties_for_mapping(self) -> list:
         return self.get_properties(
             properties, property_getter="get_properties", max_properties=15
         )
+    
+    def get_property(self, property_key: str) -> dict:
```
Comment (rytilahti) [inline miio/miot_device.py:33]: The return type should be `Dict` (import from typing is necessary) until python 3.9, iirc.

File: miio/miot_device.py (hunk 1 above)
Comment (rytilahti) [reply miio/miot_device.py:41]: Okay, fair enough. Could you improve the docstrings to contain example return values (that'll make it easy to see what to expect)?

**Comment by rytilahti** [comment]
Obsoleted by #905

**Pull request closed without merging**